pkgs/by-name/ge/geist-font/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
```

This doesn't seem like it needs a C compiler

```suggestion
, stdenvNoCC
, fetchFromGitHub
}:

stdenvNoCC.mkDerivation rec {
```

---

done